README.md

```diff
@@ -37,7 +37,7 @@ and then install necessary packages.
 
 #### a) Create a conda environment (preferred)
 
-In the repo directory
+In the "binder" directory of the main repo
```

Comment:
Can we recommend `conda env create -f binder/environment.yml` instead? Seems less error prone.

Reply:
Ya good call.